test/html.js

```diff
@@ -319,14 +319,26 @@ describe('HTML Parser', function () {
 			});
 		});
 
+		describe('#getAttribute', function () {
+			it('should return value of the attribute', function () {
+				var root = parseHTML('<p a="a1b"></p>');
+				root.firstChild.getAttribute('a').should.eql('a1b');
+			});
+
+			it('should return null when there is no such attribute', function () {
+				var root = parseHTML('<p></p>');
+				should.equal(root.firstChild.getAttribute('b'), null);
+			});
+		});
+
 		describe('#setAttribute', function () {
 			it('should edit the attributes of the element', function () {
 				var root = parseHTML('<p a=12></p>');
 				root.firstChild.setAttribute('a', 13);
 				root.firstChild.attributes.should.eql({
 					'a': '13',
 				});
-				root.firstChild.toString().should.eql('<p a=13></p>');
+				root.firstChild.toString().should.eql('<p a="13"></p>');
```

Comment:
We parse a string and convert back to string, it changed, do you think it's ok?

Reply:
Yes, browsers put double quotes around 13 when you do `setAttribute('a', 13)`.
Actually right now without `setAttribute('a', 13)` we get `<p a=12></p>`, which is not the same behaviour as in browsers, they put quotes around attribute values if it's missing during parsing, so maybe it also should be fixed.

You are free to modify my changes to expect different behaviour if you like, but in my opinion behaviour should be the same as in browsers.

Reply:
OK, I will merge this, be happy to hear different voices.